deploy/helm/listener-operator/templates/listener-classes.yaml

```diff
@@ -36,6 +41,11 @@ metadata:
   name: external-unstable
 spec:
   serviceType: NodePort
+  # Ideally we would prefer the Hostname here, but up to (including) SDP 24.7 we defaulted to the IP.
+  # We actually defaulted to Hostname for some weeks, but noticed this breaks external access to kind clusters, as the
+  # local machines can not resolve the hostnames of the nodes (such as kind-control-plane).
+  # So we stick to the old default and re-visit the default later - after SDP 24.11 is released.
+  preferredAddressType: IP
```

Comment:
This makes it sound like a temporary migration measure, but I suspect pretty heavily that we'll want to make this permanent.